ethvm/main.cpp

```diff
@@ -226,7 +246,7 @@ int main(int argc, char** argv)
 	};
 
 	executive.initialize(t);
-	if (contractCodeString != "")
+	if (!contractCodeString.empty())
```

Comment:
This condition is still the opposite. We should go create mode if the code string **is empty**.